fdbclient/admin_client.go

```diff
@@ -537,7 +502,7 @@ func cleanConnectionStringOutput(input string) string {
 func (client *cliAdminClient) GetConnectionString() (string, error) {
 	// This will call directly the database and fetch the connection string
 	// from the system key space.
-	outputBytes, err := getConnectionStringFromDB(client.fdbLibClient)
+	outputBytes, err := getConnectionStringFromDB(client.fdbLibClient, DefaultCLITimeout)
```

Comment:
Why "Default", instead of "Max", here?

Reply:
Hm, that's a good call out. I can change that.